roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -176,3 +176,12 @@ memcached_servers = {{ hostvars|ursula_memcache_hosts(groups, memcached.port) }}
 allow_headers = X-Auth-Token,X-Identity-Status,X-Roles,X-Service-Catalog,X-User-Id,X-Tenant-Id,X-OpenStack-Request-ID,X-Trace-Info,X-Trace-HMAC,OpenStack-Volume-microversion
 expose_headers = X-Auth-Token,X-Subject-Token,X-Service-Token,X-OpenStack-Request-ID,OpenStack-Volume-microversion
 allow_methods = GET,PUT,POST,DELETE,PATCH
+
+{% if barbican.enabled|bool %}
```

Comment:
we should add a new feature flag for encryption i.e cinder.encryption.enabled and use that here as well.
In addition, we must add encryption.yml to preflight-checks/tasks/ and hook it up in check_items.yml using this feature flag.  In encryption.yml should add some validators to ensure appropriate flags are set. example: brabican is enabled ..